bi_sql_editor/models/bi_sql_view_field.py

```diff
@@ -86,6 +86,12 @@ class BiSQLViewField(models.Model):
 
     index_name = fields.Char(compute="_compute_index_name")
 
+    is_many2one_clickable = fields.Boolean(
+        default=True,
```

Comment:
```suggestion
        default=False,
```
Perhaps we can set it as `False` by default, so there are no changes in current installations.

Reply:
In the meantime, I think that most of the users will want this feature.
In fact, I don't understand why put a setting here. (I don't understand when we want to put False here).
Do you have any use case ?

Reply:
@legalsylvain, There are cases where we don’t want all `many2one` fields to be clickable, so users should be able to choose which ones are navigable on the SQL view.